docs/user-guide/configuration.md

```diff
@@ -0,0 +1,63 @@
+
+---
+title: Configuration
+---
+
+vSphere Docker Volume Service consists of two components the VIB and Docker plugin. The Docker plugin requires configuration which is described in this section
+
+The Docker volume plugin at the moment supports vSphere and Photon platforms. All the volume provisioning and management operations are supported on both the platforms. Both the platforms can be used to provision VMDK based volumes and can be attached to containers. 
+
+The configurations used by a driver while performing operations are read from a JSON file and the default location where it looks for it /etc/docker-volume-vsphere.conf. You can also override it to use  a different configuration file by providing --config option and the full path to the file. Finally the parameters passed on the CLI override the one from the configuration file.
```

Comment:
Reword as "are read from a JSON file that by default is located at /etc/docker-volume-vsphere.conf"